podcasts/FolderListCell.swift

```diff
@@ -49,5 +49,6 @@ class FolderListCell: ThemeableCollectionCell {
         } else {
             unplayedBadge.isHidden = true
         }
+        unplayedBadge.updateColors()
```

Comment:
[nit] (adds breakline)

```suggestion

        unplayedBadge.updateColors()
```